tests/unit/helpers/dates/companiDates.test.js

```diff
@@ -41,11 +50,62 @@ describe('CompaniDate', () => {
   });
 });
 
+describe('GETTER', () => {
+  describe('getUnits', () => {
+    const companiDate = CompaniDatesHelper.CompaniDate('2021-11-24T07:12:08.000Z');
+
+    it('should return units', () => {
+      const result = companiDate.getUnits(['day', 'second']);
+
+      expect(result).toEqual({ day: 24, second: 8 });
+    });
+
+    it('should return empty if unit is plural or invalid', () => {
```

Comment:
Comme le resultat n'est pas vide mais plutot qu'avec les unités valides, que penses-tu plutot de "should return only valid units" ?